docker-compose.dev.yml

```diff
@@ -38,3 +42,32 @@ services:
   deck-chores:
     profiles:
       - panamax
+  logger:
+    depends_on:
+      - auth
+      - logger-postgres
+    entrypoint:
+      - /bin/bash
+      - -c
+      - |
+        until </dev/tcp/logger-postgres/5432
+        do
+          >&2 echo "PG is not available yet - sleeping"
+          sleep 1
+        done
+
+        >&2 echo "DBs are available - starting shuttle-logger"
+
+        exec /usr/local/bin/service "$${@:0}"
+  logger-postgres:
+    image: "${CONTAINER_REGISTRY}/logger-postgres:${LOGGER_POSTGRES_TAG}"
```

Comment:
Why can't we just use the stock postgres image?

Reply:
Not sure, I'll try to figure out why, and if I can't I'll remove it.

Reply:
I think we can remove the logger-postgres from the makefile as well, right? And just have it in the docker-compose.dev.yml